Temporary Compilation fix for missing plat_ipi_call_init_xlate(cpu) and plat_ipi_resched_int_xlate(cpu) which are defined in mti-malta/malta-int.c. These functions return the GIC ipi interrupt base.

These definations are expected to be present in interrupt.c w.r.t Lantiq platforms.

diff --git a/arch/mips/kernel/smp-mt.c b/arch/mips/kernel/smp-mt.c
--- a/arch/mips/kernel/smp-mt.c
+++ b/arch/mips/kernel/smp-mt.c
@@ -121,11 +121,19 @@
 
 	switch (action) {
 	case SMP_CALL_FUNCTION:
+#ifdef CONFIG_MIPS_MALTA
 		gic_send_ipi(plat_ipi_call_int_xlate(cpu));
+#else
+/*Need to implement [platform specific thing and that function has to go here*/
+#endif
 		break;
 
 	case SMP_RESCHEDULE_YOURSELF:
+#ifdef CONFIG_MIPS_MALTA
 		gic_send_ipi(plat_ipi_resched_int_xlate(cpu));
+#else
+/*Need to implement [platform specific thing and that function has to go here*/
+#endif
 		break;
 	}
 
